assets/data/asset_no_extension

```diff
@@ -0,0 +1,3 @@
+CustomAsset (
+    value: 13
+)
```

Comment:
```suggestion
)

```

Reply:
Then we don't get a complaint about a file not ending with a new line in the Github UI or elsewhere.